inventory.md

```diff
@@ -736,6 +774,25 @@ This section contains references which are relevant to the requirements expresse
     * **Publication date:** 2024
   </details>
 
+* [CISA SBOM Community - SBOM Reference Implementations](https://github.com/SBOM-Community/SBOM-Generation) - A white paper examinining the practical challenges of producing robust, National
+Telecommunications and Information Administration (NTIA) Minimum Elements-adherent,
+Software Bills of Materials (SBOM) that not only meet the NTIA Minimum Elements
+but can go beyond this to meet future compliance frameworks. As of publication,
+the authors' research found that no single open source tool can reliably generate
+an SBOM that adheres to NTIA Minimum Elements out of the box. The authors propose
+a six-step process that separates SBOM creation (or "authoring") into distinct,
+manageable phases (Generation, Augmentation, Enrichment, Verification, Signing).
```

Comment:
I see only 5 phases listed here, yet also see `six-step process`. Did you forget to add "Document Linking"?